app/views/measures/_measure.html.erb

```diff
@@ -64,7 +68,10 @@
 
   <td class="footnotes-col govuk-table__cell govuk-table__header--numeric">
     <% if measure.has_measure_footnotes? %>
-      <%= link_to measure.footnotes.size == 1 ? measure.footnotes.first.code : "Footnotes",  "#", class: 'reference', 'data-popup-ref' => "#{measure.destination}-#{measure.id}-footnotes", "aria-label": "Click to open a dialog with the footnotes for this measure", role: "button" %>
+      <div data-controller="anchor">
+        <%= link_to measure.footnotes.size == 1 ? measure.footnotes.first.code : "Footnotes", "#", class: 'reference', data: { action: "click->anchor#launchModal", modal_ref: "#{measure.destination}-#{measure.id}-footnotes" }, "aria-label": "Click to open a dialog with the footnotes for this measure", role: "button" %>
+        <div id="modal" class="modal tariff-info" data-controller="modal" data-anchor-target="modal"></div>
```

Comment:
and again

Reply:
This modal div must sit within the anchor div which cannot move outside the scope of the row cell.